core/src/main/java/org/apache/iceberg/rest/CatalogHandlers.java

```diff
@@ -89,6 +90,11 @@ public CommitFailedException wrapped() {
     }
   }
 
+  public static RESTCatalogConfigResponse fetchServerConfig() {
```

Comment:
What is the purpose of updating `CatalogHandlers`? Since this doesn't actually do translation to the `Catalog` API, it doesn't seem like something we should provide a helper for. Plus, this doesn't give the caller any opportunity to customize the config.

Reply:
The tests fail if there's not something in place of this call, since we call `initialize` and not the older `initializeInternal`.

So something needs to be returned here.